src/deque.py

```diff
@@ -0,0 +1,94 @@
+import sys
```

Comment:
No docstring in the module, and you don't use sys.

Please clean up the linter errors